app/views/kudos/index.html.erb

```diff
@@ -20,3 +21,23 @@
     <% end %>
   <% end %>
 </div>
+<% end %>
+<% if @kudos %>
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts('Kudos History') %></h2>
+<!--/descriptions-->
+<!--main content-->
+<% if logged_in? && @kudos.present? %>
```

Comment:
I am not convinced, I have memories that I checked and I could see others kudos.